api/SVGAltGlyphDefElement.json

```
@@ -24,10 +26,12 @@
             "version_added": false
           },
           "opera": {
-            "version_added": null
+            "version_added": "≤15",
```

```suggestion
            "version_added": "15",
```

I confirmed no support in Opera 12.16 using http://mdn-bcd-collector.appspot.com/tests/api/SVGAltGlyphDefElement

I'm actually not 100% sure this means no support of the `<altGlyphDef>` element, but the interface isn't there. Actually neither is `SVGAltGlyphElement`. There's always something more wrong :)